README.md

```diff
@@ -32,3 +32,19 @@ brew link qt --force
 patch ./3rdparty/sunxi-tools/fel_lib.c < ./3rdparty/sunxi-tools.diff
 make
 ```
+
+
+# Execute
+
+## Linux
+
+```
+build/hakchi-gui
```

Comment:
`build/hakchi-gui` without a leading dot works. A leading dot is needed only when the command lies in the current folder.

This is so that the shell can tell a part between a command that could be in the path or in the current directory. For instance if we type in `foobar` the shell would not know where to look.

If a command has a slash anywhere there's no confusion for the shell. A leading slash means an absolute path. Staring with a dot from here, if it starts with .. it goes up to the parent folder and with a tilde (~) to the users account or another user account.

TL;DR if paths have at least a slash all is fine, this is why we use `./foo` to tell the shell that it is in the current folder